src/FakeItEasy/Core/FakeWrapperConfigurator.cs

```diff
@@ -38,9 +38,9 @@ public override IFakeOptionsForWrappers<T> Wrapping(T wrappedInstance)
             return this.fakeOptions.Wrapping(wrappedInstance);
         }
 
-        public override IFakeOptions<T> WithAdditionalAttributes(IEnumerable<CustomAttributeBuilder> customAttributeBuilders)
+        public override IFakeOptions<T> WithAdditionalAttributes(params Expression<Func<Attribute>>[] customAttributeExpressions)

```

Comment:
Is `customAttributeExpressions` the best name? It feels a little like naming a `string` parameter `somethingString`. 

Usually I'd name a `Func<T>` parameter like a method, e.g. `createAttribute`, so when I invoke it I can write

```C#
var attribute = createAttribute();
```

but I'm not sure how that naming convention would translate to a collection of such objects. I also noticed that in `IProxyOptions` the property is named `AdditionalAttributes`, so perhaps even just `attributes` will work?